plugins/Goals/Archiver.php

```diff
@@ -128,59 +127,79 @@ public function aggregateDayReport()
         }
     }
 
+    private function hasAnyGoalOrEcommerce($idSite)
```

Comment:
above in line 120 it should be maybe `$this->usesEcommerce($this->getSiteId())` instead of `if (Manager::getInstance()->isPluginActivated('Ecommerce')) {` ?

Reply:
That's actually on purpose. So it will create an empty archive if the plugin is activated, but not used for the current site.